### PR TITLE
Create dummy login credentials for Demo #45

### LINKED ISSUE #45: Create dummy login credentials for Demo
* Currently website configured for a live database
* It is not possible to have a demo of the website with static dummy credentials
- [x] Create dummy credentials for login purpose
   * User email: admin@email.com
   * Password: admin
- [x] Restructure database fetch requests - void any such requests in case of login with dummy credentials
- [x] Update the readme with the new login information 
- [x] Link the fix of this issue with issue #43

### PR DESCRIPTION
**Pull request by anand-nakat**
Fixes #45 
 Created Static User to access the Index Page. It was not possible previously, as the access was only allowed only to registered user. But since database is yet to be configured, it wasn't possible to access index page to anyone.

image

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 CyanoKhoj is a web-app which employs the citizen science program by tracking their tweets and subsequent data analysis on Google Earth Engine, aimed at effective CyanoHAB detection and monitoring in water bodies around the world. The objective is to identify potential CyanoHAB infested locations based on Twitter data analysis and location extraction and further analyze these locations using satellite data in Google Earth Engine.
 * [Web-app Link](http://34.67.7.17/CyanoKhoj/)
-  * Username: admin
+  * Email ID: admin@email.com
   * Password: admin
```
Comment (anand-nakat) [inline README.md:17]: Credentials updated in README.md

File: login.php [hunk 2]
```diff
@@ -79,7 +79,7 @@
 						</div>
 						
 						<div>
-							<a href="#" class="txt1" style="color: #DD9106; font-size:2rem;" onclick="alert('Please login with username: admin & password: admin for demo.')">
+							<a href="#" class="txt1" style="color: #DD9106; font-size:2rem;" onclick="alert('Please login with email: admin@email.com & password: admin for demo.')">
```
Comment (anand-nakat) [inline login.php:82]: Credentials updated for Demo Button Alert

**Review by Chintan2108 — CHANGES_REQUESTED** [review]
@anand-nakat Looks good to me. 
You can get rid of the console.log statements.

File: index.php [hunk 3]
```diff
@@ -509,14 +509,12 @@ function initMap() {
 $(document).ready( function () {  
 
         const userId=<?php echo($_SESSION['user_id']); ?>;
-        console.log(userId);
         
         //Lock Input Field By Default as Modal is Opened    
         $("#my-profile").click(function () {
-          console.log("Modal Opened");
+           ("Modal Opened");
```
Comment (anand-nakat) [inline index.php]: I'm fixing this in new commit

**Review by Chintan2108 — APPROVED** [review]
Good Work. PR credit subject to successful testing on GCP.

**Comment by Chintan2108** [comment]
give @anand-nakat 75 points

**Pull request merged**